checks/check-app-health.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"net/http"
+	"io/ioutil"
+    "strconv"
+    "log"
+
+	maps "github.com/ashwanthkumar/golang-utils/maps"
+	"github.com/gambol99/go-marathon"
+)
+
+type AppHealth struct{}
+
+func (h *AppHealth) Name() string {
+	return "apphealth"
+}
+
+// Checks App health status of App Endpoint by reading router.hosts 
+//  and Check for 200 status code
+func (h *AppHealth) Check(app marathon.Application) AppCheck {
+	host := maps.GetString(app.Labels, "router.hosts", "notDefined")
+    host_path  := app.HealthChecks[0].Path
+	host_url := strings.Join([]string{"http://", host, host_path}, "")
+    result := Pass
+    message := fmt.Sprintf("HTTP Status OK!!")
+
+	resp, err := http.Get(host_url)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	responseString := string(body)
+
+
+	if resp.StatusCode >= 200 && resp.StatusCode <= 299 {
+		result = Pass
+	} else {
+		result = Critical
+		message = fmt.Sprintf("HTTP Response Status: ,  " + strconv.Itoa(resp.StatusCode), responseString)
```

Comment:
Why using `+` when using `fmt.Sprintf`?

Reply:
Modified the sprintf format https://github.com/ashwanthkumar/marathon-alerts/pull/35/commits/b8c60533f9dc30c07b399c4ddf6e2b798bf895d1